src/fileio/datafile.cxx

```diff
@@ -902,6 +902,33 @@ void Datafile::add(Vector3D &f, const char *name, bool save_repeat) {
   }
 }
 
+namespace {
+// Read a value from file and check it matches reference_value, throw if not
+void checkGridValue(DataFormat* file, const std::string& name,
+                    const std::string& filename, const int reference_value) {
```

Comment:
unneeded `const` on `int`?

Reply:
Makes no different to the signature, but the local variable is `const` :)